packages/opentelemetry-instrumentation-anthropic/tests/test_completion.py

```diff
@@ -107,12 +107,121 @@ def test_anthropic_multi_modal(exporter):
+        max_tokens=1024,
+        messages=[
+            {
+                "role": "user",
+                "content": "Tell me a joke about OpenTelemetry",
+            }
+        ],
+        model="claude-3-haiku-20240307",
+        stream=True,
+    )
+
+    for _ in response:
+        pass
+
+    spans = exporter.get_finished_spans()
+    assert [span.name for span in spans] == [
+        "anthropic.completion",
+    ]
+    anthropic_span = spans[0]
+    assert (
+            anthropic_span.attributes["llm.prompts.0.user"]
+            == "Tell me a joke about OpenTelemetry"
+    )
+    assert (anthropic_span.attributes.get("llm.completions.0.content"))
+    assert anthropic_span.attributes["llm.usage.prompt_tokens"] == 8
+    assert (
+            anthropic_span.attributes["llm.usage.completion_tokens"]
+            + anthropic_span.attributes["llm.usage.prompt_tokens"]
+            == anthropic_span.attributes["llm.usage.total_tokens"]
+    )
+
+
+@pytest.mark.vcr
+@pytest.mark.asyncio
+async def test_async_anthropic_message_create(exporter):
+    client = AsyncAnthropic()
+    response = await client.messages.create(
+        max_tokens=1024,
+        messages=[
+            {
+                "role": "user",
+                "content": "Tell me a joke about OpenTelemetry",
+            }
+        ],
+        model="claude-3-opus-20240229",
+    )
+
+    spans = exporter.get_finished_spans()
+    assert [span.name for span in spans] == [
+        "anthropic.completion",
+    ]
+    anthropic_span = spans[0]
+    assert (
+            anthropic_span.attributes["llm.prompts.0.user"]
+            == "Tell me a joke about OpenTelemetry"
+    )
+    assert (
+            anthropic_span.attributes.get("llm.completions.0.content")
+            == response.content[0].text
+    )
+    assert anthropic_span.attributes["llm.usage.prompt_tokens"] == 8
+    assert (
+            anthropic_span.attributes["llm.usage.completion_tokens"]
+            + anthropic_span.attributes["llm.usage.prompt_tokens"]
+            == anthropic_span.attributes["llm.usage.total_tokens"]
+    )
+
+
+@pytest.mark.vcr
+@pytest.mark.asyncio
+async def test_async_anthropic_message_streaming(exporter):
+    client = AsyncAnthropic()
+    response = await client.messages.create(
+        max_tokens=1024,
+        messages=[
+            {
+                "role": "user",
+                "content": "Tell me a joke about OpenTelemetry",
+            }
+        ],
+        model="claude-3-haiku-20240307",
+        stream=True,
+    )
+
+    async for _ in response:
+        pass
+
+    spans = exporter.get_finished_spans()
+    assert [span.name for span in spans] == [
+        "anthropic.completion",
+    ]
+    anthropic_span = spans[0]
+    assert (
+            anthropic_span.attributes["llm.prompts.0.user"]
+            == "Tell me a joke about OpenTelemetry"
+    )
+    assert (anthropic_span.attributes.get("llm.completions.0.content"))
```

Comment:
Can we test that this attribute is actually the response we accumulated in line 209?